crawler/file_handler.py

```
@@ -124,6 +260,115 @@ def extract_text(path: Path, ext: str) -> tuple[str, str]:
+            log.warning(f"  외부 이미지 개수 제한({_MAX_INLINE_IMAGES}개) 초과, 나머지 건너뜀")
+            break
+        if url in urls_seen:
+            continue
+        urls_seen.add(url)
+
+        parsed_path = Path(urlparse(url).path)
+        raw_ext = parsed_path.suffix[1:].lower() if parsed_path.suffix else "jpg"
+        if raw_ext not in ("jpg", "jpeg", "png", "gif", "webp", "svg"):
+            raw_ext = "jpg"
+
+        mime_type = f"image/svg+xml" if raw_ext == "svg" else f"image/{'jpeg' if raw_ext == 'jpg' else raw_ext}"
```

Ruff (0.15.12)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)